badge/README.md

```diff
@@ -0,0 +1,154 @@
+# Badge components
+
+## Badge design specs
+
+You can find the design specs on [decathlon.design](https://www.decathlon.design/).
+
+## Usage
+
+If you want to use components of this module in your android mobile application, you should
+first add the Gradle dependency in your Gradle file:
+
+```kotlin
+implementation("com.decathlon.vitamin.compose:badge:<versions>")
+```
+
+### VitaminCountBadges
+
+```kotlin
+object VitaminCountBadges {
+    @Composable
+    fun Accent(
+        count: Int,
+        badgeSize: BadgeSize = VitaminBadgeSizes.smallSize(),
+        badgeStyle: BadgeStyle = VitaminBadgeStyle.accentStyle(badgeSize = badgeSize),
+        nbMaxChar: Int = defaultMaxChar,
+        content: @Composable BoxScope.() -> Unit
+    ) {
+        VitaminCountBadge(
+            count = count,
+            badgeStyle = badgeStyle,
+            nbMaxChar = nbMaxChar,
+            content = content,
+            badgeSize = badgeSize
+        )
+    }
+}
+```
+
+The minimal usage of the component is the count of the badge and the content (Image or other composable). 
+Customize the size with a small, medium or large size variants
+Customize the badge style with BadgeStyle object
+Select the number of digit that you want before add "+" (nbMaxChar = 2, count= 110, result = 99+ )
+
+```kotlin
+VitaminCountBadges.Default(count = 5){
+    //YOUR COMPOSABLE CONTENT
+}
+```
+
+Parameters | Descriptions
+-- | --
+`count: Int`| the text inside de badge
+`badgeSize: BadgeSize` | the size of the badge,
+`badgeStyle: BadgeStyle` | the style of the badge 
+`nbMaxChar: Int` | the number of digit that you want before add "+" (nbMaxChar = 2, count= 110, result = 99+ )
+`content: @Composable BoxScope.() -> Unit`| the icon or content which own the badge
+
+### VitaminCountBadges All Variants
+
+There are multiple variants of the badge with the same parameters:
+
+* `Brand`
+* `Default`
+* `Reversed`
+* `Alert`
+
+### VitaminCountBadges without content
+
+```kotlin
+object VitaminCountBadges {
+    @Composable
+    fun Accent(
+        count: Int,
+        badgeSize: BadgeSize = VitaminBadgeSizes.smallSize(),
+        badgeStyle: BadgeStyle = VitaminBadgeStyle.accentStyle(badgeSize = badgeSize),
+        nbMaxChar: Int = defaultMaxChar,
+        isHiddenCountLeft: Boolean = false
```

Comment:
I'm not sure to understand the difference between `nbMaxChar` and `isHiddenCountLeft` :thinking:

Reply:
both are to format text but : 
NbMaxChar transform 130 to : (99+) 
<img width="143" alt="Capture d’écran 2022-06-17 à 13 36 10" src="https://user-images.githubusercontent.com/16080757/174290908-c926f271-bfe0-4292-9099-3bfa1a661bc4.png">
413a95.png">
IsHiddenCOuntLeft (changed into AreHiddenItemsLEft) transform 4 to : (+4) 
<img width="375" alt="Capture d’écran 2022-06-17 à 13 36 02" src="https://user-images.githubusercontent.com/16080757/174290892-32d0e217-df22-4fe1-945c-fd7e15